OWNERS.md

```diff
@@ -79,6 +79,9 @@ contributors to envoy-setec and relevant Slack channels from:
   * Alessandro Gario ([alessandrogario](https://github.com/alessandrogario))
   * Mike Myers ([mike-myers-tob](https://github.com/mike-myers-tob))
 
+* [Feb 2022 Security Release Contributors]
+  * Pradeep Rao ([pradeepcrao](https://github.com/pradeepcrao)) expiring 2/28/2022
```

Comment:
2022 or 2021? Either is good, since I believe @pradeepcrao will be contributing valuable security fixes over the next year.

Reply:
2021 would be in the past, right?

Reply:
Sorry, brain fart, I think we should make it 2023 then.

Reply:
Done.

Reply:
I think if we're making it 2023 we just add him to the setec group, as discussed on #envoy-security-team.  Ditto @RyanTheOptimist as long as we have the file open?

Reply:
/wait

Reply:
Done